pkg/dom0-ztools/Dockerfile

```diff
@@ -42,3 +43,13 @@ FROM scratch
 COPY --from=zfs /out/ /
 # hadolint ignore=DL3020
 ADD rootfs/ /
+RUN touch /etc/group && touch /etc/passwd \
+    && addgroup -g 0 root \
+    && adduser -S -D -H -h /root -s /bin/sh -G root -u 0 root
```

Comment:
```suggestion
RUN touch /etc/group /etc/passwd
RUN addgroup -g 0 root
RUN adduser -S -D -H -h /root -s /bin/sh -G root -u 0 root
```

Reply:
@christoph-zededa AFAIR the linter will complain about that and suggest to use a single RUN

Reply:
Hmm, okay :-(

Reply:
fixed.